firefox_puppeteer/ui/toolbars.py

```diff
@@ -347,22 +347,42 @@ class IdentityPopup(BaseLib):
 
     @property
     def box(self):
+        """ Provides access to the identity-box element.
```

Comment:
It's not redundant but it will cause broken documentation. The function will be marked to not return any value if you remove the `:returns:` line.